recipes/sasview/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "sasview" %}
+{% set version = "v5.0_beta.2_8_g390494d7c" %}  # git describe --tags --dirty --always | sed 's/-/_/g'
+{% set file_ext = "tar.gz" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  git_url: https://github.com/SasView/sasview
+  git_rev: 390494d7cc6099973fce721a89afc666ae254c11
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - numpy
+    - pyqt >=5
+    - python
+    - setuptools
+  run:
+    - bumps
+    - h5py >=2.5
+    - lxml
+    - matplotlib
+    - numpy
+    - periodictable >=1.5.0
+    - pillow
+    - pyparsing >=2.0.0
+    - pyqt >=5
+    - python
+    - qt5reactor
+    - sasmodels
+    - scipy
+    - twisted <=18.4
+    - xhtml2pdf
+
+test:
+  imports:
+    - sas.sasview
+    - sas.qtgui
+    - sas.qtgui.MainWindow
+  commands:
+    - which sasview
+
+about:
+  home: https://github.com/SasView/sasview
+  license: Custom
+  license_family: AGPL
+  license_url: https://github.com/SasView/sasview/blob/master/LICENSE.TXT
+  summary: SasView is a Small Angle Scattering (SAS) analysis package
+
+extra:
+  recipe-maintainers:
```

Comment:
@mdoucet @rozyczko @stuartcampbell Are you all happy to be listed as maintainers of this recipe?

Reply:
@mdoucet @rozyczko @stuartcampbell Friendly poke.

Reply:
@CJ-Wright @chrisburr @mrakitin 
SasView 5.0 is in pre-release (early beta) stage at this point. Not really ready for proper conda-forge release. It would be better to put SasView 4.2.1 (based on python 2.7 and wx-Python) instead, at least until 5.0 is out as a full release.

Reply:
Sounds good. Regarding the release version, I have no issue creating a recipe for 5.0 if it's properly documented as a beta, but I think we should definitely create a recipe for 4.2.1, which is the latest official release.

Reply:
Agreed with @mdoucet. If the recipe is documented as a beta, I have no problems with it, given we also have 4.2.1 as official.

Reply:
@rozyczko, @mdoucet, thanks for the information. The motivation for this PR was that @stuartcampbell had a conversation with your team and mentioned you were enthusiastic about having the package at conda-forge. We currently have it in our conda channel (https://anaconda.org/lightsource2-tag/sasview), and as you can see there, the name clearly states it's beta: `v5.0_beta.2_8_g390494d7c`. I'm fine with postponing making changes to this PR until the stable version is released.

Regarding the 4.2.1 recipe, I am not sure I can help here as Python 2.7 will die soon, and our facility is 3.6+ only.